locations/spiders/sportsdirect.py

```diff
@@ -0,0 +1,69 @@
+import json
+import re
+
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class SportsDirectSpider(scrapy.Spider):
+    name = "sportsdirect"
+
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    available_countries = [
+        "MY",
+        "FR",
+        "AT",
+        "CY",
+        "DE",
+        "EE",
+        "KW",
+        "IE",
+        "LU",
+        "PL",
+        "SI",
+        "CZ",
+        "GB",
+        "AE",
+        "IS",
+        "HU",
+        "ES",
+        "LV",
+        "SK",
+        "BE",
+        "LT",
+        "PT",
+        "SA",
+        "RW",
+    ]
```

Comment:
Instead of hardcoding these could we fetch them from https://www.sportsdirect.com/stores `//select[@id="storeFinderCountryDropDown"]/option/@value`?

Reply:
Also just found [this](https://www.sportsdirect.com/stores/all) if it's easier

Reply:
The other brands have the same url:
https://www.fieldandtrek.com/stores/all
https://www.lillywhites.com/stores/all